### ISSUE TITLE
Breaking in Angular 1.4

### ISSUE THREAD

**Issue by jdchizzle** [revolunet/angular-carousel#338]
Ran a bower update and all carousel functions seem to stop working. After downgrading to Angular 1.2.28, everything works fine again. I'll see if I have some time to look into the issue. Nothing seems to come up on the error log at all.

**Comment by sckoh** [comment]
+1

nothing show up

**Comment by malfborger** [comment]
Is there a timeline for support for 1.4?

**Comment by jbarros35** [comment]
any workaround????

**Comment by revolunet** [comment]
Hey. No much time to work on this currently, sorry about that :/

My first guess is to check when happens here : https://github.com/revolunet/angular-carousel/blob/master/src/directives/rn-carousel.js#L161-L167
is the ng-repeat expression correctly parsed and valid ?

PR much appreciated (this repo is looking for contributors :))

**Comment by jdchizzle** [comment]
Dove into this and wasn't quite able to isolate the issue. Running with Google's CDN for Angular 1.4.1 (and the corresponding touch dependency), the carousel seems to work perfectly fine. Was anyone else able to isolate this issue at all?

**Comment by jbarros35** [comment]
I don't have enough knowledge on this for giving some help, most of us does not. We are javascript programmers not Angular engineers. By the way, I cannot test my application using Karma and Jasmine because its stuck on dependencies.

**Comment by revolunet** [comment]
I'm just a developer too :)
What are the step to reproduce ? simply `bower update`?

**Comment by cedricblondeau** [comment]
Hi,

I just updated my app from Angular 1.2.8 to Angular 1.4.3 and the library works fine (angular-carousel 0.3.12).

If nothing shows up and if there is no JS errors after a simple `bower update`, I suggest that the library itself (angular-carousel) may have been upgraded as well (from 0.2.x to 0.3.x).
Since version 0.3.0, `li` elements are absolutely positioned and the `ul` element `height` CSS property has to be set manually. If you don't do so, the `ul` just won't show up. (see issue #309).

**Comment by revolunet** [comment]
Thanks @cedricblondeau

**Comment by dkirsanov** [comment]
Did someone hit the issue when swiping the pictures using cursor doesn't work? No problems with a touch swipe, only doing that by mouse.

**Comment by dkirsanov** [comment]
If someone hit the issue described above, here is the solution:
`ul[rn-carousel] > li > img {
    -webkit-user-drag: none;
}`

**Comment by mhipo1364** [comment]
:+1:

**Comment by blockjon** [comment]
I'm using angular 1.4.5... the carousel does not work at all.

**Comment by anatoly314** [comment]
same issue as @blockjon , absolutely empty screen nothing being rendered

**Comment by kross-f** [comment]
@anatoly314  @blockjon Did you try adding height to <ul> ? 

@cedricblondeau's fixed that for me (nothing rendered, no visible errors).

**Comment by anatoly314** [comment]
@kross-f , yes it was the issue. It wasn't clear from the manual that I should do it.

**Comment by zeeshanjan82** [comment]
Thanks @cedricblondeau you made my day. I was trying on angular 1.4 and after setting height everything worked fine

**Comment by jdchizzle** [comment]
+1 @kross-f Worked for me as well.
